.gitlab/ci/on-push.yml

```diff
@@ -37,6 +37,42 @@ trigger-private-build:
   - ./Documentation/commonServerDocs.sh
   - section_end "Create Release Notes and Common Server Documentation"
 
+stop-running-pipelines:
+  tags:
+    - gke
+  stage: unittests-and-validations
+  needs: []
+  extends:
+    - .default-job-settings
+  variables:
+    master_branch_name: master
+    CONTENT_PIPELINES_API_URL: https://code.pan.run/api/v4/projects/2596/pipelines # disable-secrets-detection
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push)$/ && $CI_COMMIT_BRANCH != $master_branch_name'
+  script:
+    - section_start "Stop running piplines on current branch"
```

Comment:
please move the script to a new file `Utils/gitlab/stop_running_pipelines.sh` (accept GITLAB_CANCEL_TOKEN and CI_COMMIT_BRANCH as args, other vars define there).

(reason: we want to avoid the practice of writing scripts directly in the yml, in order to be able to run them locally. see similar issue: https://jira-hq.paloaltonetworks.local/browse/CIAC-5423)